docs/developer/pull-requests.md

```diff
@@ -0,0 +1,128 @@
+
+### PR assignment
+
+Most of the time, there are other folks involved in the project/feature you are working on, even if it was only during the design phase. Add all of them. If you have on one or your PR is just a small fix/random improvement you came up with, use the GitHub suggested reviewers. If you are in doubt, ping the team in Slack and ask for help.
+
+### Review comments
+
+In general, all comments are important and should be addressed and/or acknowledged before the PR is merged (emoji reaction, comments, etc.).
+
+If you are planning on not addressing one, talk to the reviewer who wrote it and let them know.
+
+If you are requested to make a change that you would like for some reason to not add to the PR and do it later, talk to the reviewer. But, please don’t use it as an excuse to skip tests. And create issues for it. Checklists in parent issues are super useful for this scenario.
+
+## As a reviewer
+
+### Time to first review SLA
+
+24 hours, from the time the PR is open to the time the first review is submitted. That should be more than enough to take into account timezones and PRs open at the end of the day. It gives us at least a full working day to get to it.
+
+If you think you won't able to meet this, talk to the author.
+
+### PR size
+
+If you think the PR is too big and/or could be split in smaller chunks in order to make the review easier, talk to the author. It is acceptable to ask for a PR to be split before moving forward with the review.
+
+### PR assignment
+
+If you don’t think you should be in a particular PR or you don’t think you will have the time to do it, talk to the author so they can find a replacement if needed. Don’t let it sit, if you are assigned, you are still responsible for reviewing it. The GitHub notifications section is very useful to check if you are assigned to a PR.
+
+### Review comments
+
+Making your intent as a reviewer more clear helps the author decide on the next steps, prioritizing their work and avoid future back and forth:
+
+* Whenever it’s something you don’t need to be changed in order to approve the PR, prepend it with “Nit:”. It’s fast, short and everyone understand it. If it’s not necessarily a suggested change and it’s purely educational, make that clear as well. When possible, use the "Add a suggestion" feature. This makes it easier to deal with spelling, one-time-use variable naming, grammar, small logical adjustments, etc.
+* If you are ok with resolving one or your comments in a future PR, let the author know. This one is a bit tricky because in general, we don’t want comments to be left for later, if we write them is for a reason. But it’s true that there are times where what we ask for is too big and/or complex and will make more damage to the PR than good to the codebase. If that’s the case and there is no risk implementing it later, let the author know.
+`TODO: add some examples for what is ok to postpone and what is not`
```

Comment:
@chrisdoherty4 looking at you here since you proposed this
I was thinking one clear example could be unit tests (we don't postpone unit tests unless the code is under a feature toggle)
Anything else?

Reply:
I think unit tests is a bit of a special case. Some people do the whole "I'll write them later" but frankly I think there's rarely, if ever, a good reason to do that. 

Beyond unit tests, its really up to people to discuss it with the reviewer. Does that resonate with you?

Reply:
Yeah, sounds good. I was only following up on this since you mentioned before we should probably add some examples about what's ok to postpone and what's not.
I added a little bit about unit tests, let me know.

Reply:
lgtm